admin_manual/configuration_server/email_configuration.rst

```diff
@@ -196,10 +195,9 @@ which uses the port 465/tcp:
     "mail_smtphost"     => "smtp.server.dom:465",
     "mail_smtpsecure"   => 'ssl',
 
-TLS
-^^^
-A secure connection will be initiated using the STARTTLS protocol which
-uses the default port 25/tcp:
+STARTLS
```

Comment:
Outsch, good eyes. Too many SSL/TLS/STARTTLS in here :-)

Thanks for the review!!